docs/howto/kubernetes/gardener/hibernate-shoot-cluster.md

```diff
@@ -0,0 +1,73 @@
+---
+description: How to hibernate a Gardener-based Kubernetes cluster
+---
+# Hibernating a Kubernetes cluster
+
+There will be times when you won't be using your Gardener-based cluster 
+much, if at all. To save on costs, you can put the whole cluster in 
+hibernation. If you do, then from that time on, and until you wake the 
+cluster up, you will be paying *only* for the total storage space your 
+cluster occupies. Let us see how you can hibernate a cluster from the 
+{{gui}}, and then how you can wake up an already hibernated cluster.
+
+## Assumptions
+
+We assume you have already used Gardener in {{brand}} to spin up a 
+Kubernetes cluster, which is now humming away. If you've never done 
+this before, please feel free to [follow this 
+guide](../create-shoot-cluster).
+
+## Hiberneting a cluster
+
+Fire up your favorite web browser and navigate to 
+<https://{{gui_domain}}>. Make sure the vertical pane at the left-hand 
+side of the page is in full view, then choose 
+*Containers → [{{k8s_management_service}}](https://{{gui_domain}}/containers/gardener)*.
+In the main pane, you will see your Gardener cluster.  Click anywhere
+on the corresponding row for a detailed view of the various cluster
+characteristics. Go to the *Status* tab and check if hibernation is
+possible.
+
+![Gardener cluster status](assets/garhiber-01.png)
```

Comment:
This screenshot contains markers labeled ① ② ③, but these numbers are not referenced in the prose. Please remove them from the screenshot.